src/Server/TCPHandler.cpp

```diff
@@ -1244,6 +1245,9 @@ void TCPHandler::sendProfileEvents()
 
         state.profile_events_block_out->write(block);
         out->next();
+
+        LOG_TRACE(log, "Sending profile events block with {} rows, {} bytes took {} milliseconds",
```

Comment:
would't it be too talkative?

Reply:
In one of the test runs I see
```
davenger@davenger-xps:~/Downloads$ zstdcat clickhouse-server\ \(7\).log.zst | wc -l
1790422
davenger@davenger-xps:~/Downloads$ zstdcat clickhouse-server\ \(7\).log.zst | grep -a 'Sending profile events' | wc -l
46666
```

Do you think it makes sense to move it under smth like `if (elapsed_milliseconds > 100)` ?

Reply:
yeah, most likely it is useless if the elapsed time is small